crates/vhost-user-backend/Cargo.toml

```diff
@@ -8,16 +8,16 @@ edition = "2018"
 license = "Apache-2.0"
 
 [dependencies]
-libc = "0.2.39"
+libc = "0.2.137"
```

Comment:
Does `vhost-user-backend` really need the latest version of `libc`?

Given that SemVer allows a dependency specified with caret version to be satisfied by the given version or any later version within the API context (for zero versions, any version with the same minor number), I think we should play conservative here. Otherwise, we could be making the distros packaging work a lot harder and imposing an arbitrary version to the consumers of the crate.

Reply:
Good plan. I updated the PR so it only touches the rust-vmm dependencies.